seacrowd/sea_datasets/sib_200/sib_200.py

```diff
@@ -0,0 +1,204 @@
+# coding=utf-8
+# Copyright 2024 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+SIB-200 is the largest publicly available topic classification dataset based on Flores-200 covering 205 languages and dialects.
+The train/validation/test sets are available for all the 205 languages.
+"""
+
+from typing import List, Tuple, Dict
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@misc{adelani2023sib200,
+      title={SIB-200: A Simple, Inclusive, and Big Evaluation Dataset for Topic Classification in 200+ Languages and Dialects},
+      author={David Ifeoluwa Adelani and Hannah Liu and Xiaoyu Shen and Nikita Vassilyev and Jesujoba O. Alabi and Yanke Mao and Haonan Gao and Annie En-Shiun Lee},
+      year={2023},
+      eprint={2309.07445},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DATASETNAME = "sib_200"
+
+_DESCRIPTION = """\
+SIB-200 is the largest publicly available topic classification dataset based on Flores-200 covering 205 languages and dialects.
+The train/validation/test sets are available for all the 205 languages.
+"""
+
+_HOMEPAGE = "https://github.com/dadelani/sib-200"
+
+_LANGUAGES = [
```

Comment:
Please only use the list of language IDs for `_LANGUAGES` for consistency, but the subset names should retain the transcript types as well.

Reply:
Done